File: createll.java
import java.io.*;
class LinkedList
{
    
static class Node
{
  
  int data;
  Node next;
  Node(int data)
  {
    this.data=data;
    this.next=null;
  }
}

static Node push(Node head, int data)
{
    Node node = new Node(data);
    node.data=data;
    node.next=head;
    head=node;
    return node;
}

static void traverse(Node head)
{
    while(head!=null)
    {
        
        System.out.println(head.data);
        head=head.next;
    }
}

public static void main(String gg[])
{
    Node head=null;
    head=push(head, 1);
    head=push(head, 2);
    head=push(head, 3);
    head=push(head, 4);
    traverse(head);
}

}

File: createll.cpp
#include<bits/stdc++.h>
using namespace std;

class Node
{
  public:
  int data;
  Node* next;
  Node(int data)
  {
      this->data=data;
      this->next=NULL;
  }
};

void push(Node** head, int data)
{
    Node* node = new Node(data);
    node->data=data;
    node->next=(*head);
    (*head)=node;
}

void traverse(Node* head)
{
    while(head!=NULL)
    {
        cout<<head->data<<endl;
        head=head->next;
    }
}

int main()
{
    Node* head=NULL;
    push(&head, 1);
    push(&head, 2);
    push(&head, 3);
    push(&head, 4);
    traverse(head);
    return 0;
}